src/chrome/content/rules/ShopToIt.ca.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+	Connection refused:
+		- retailers.shoptoit.ca
+		- smallbusiness.shoptoit.ca
+
+	Invalid certificate:
+		- shoptoit.ca, equivalent to www.shoptoit.ca
+		- fr.shoptoit.ca
+		- listingsca.shoptoit.ca, equivalent to www.shoptoit.ca
+		- yahoo.shoptoit.ca
+-->
+
+<ruleset name="ShopToIt.ca">
+	<target host="shoptoit.ca" />
+	<target host="www.shoptoit.ca" />
+	<target host="listingsca.shoptoit.ca" />
+	<target host="merchants.shoptoit.ca" />
+
+	<rule from="^http://(listingsca\.)?shoptoit\.ca/"
+		to="https://www.shoptoit.ca/" />
```

Comment:
This does not work for me on deep links.

Reply:
Could you give an example link? It looks to me like everything on `listingsca.shoptoit.ca` gets 301'd to `www.shoptoit.ca` by the web server itself.

Reply:
`http://listingsca.shoptoit.ca/library/reviews-top-rated-gps/2399`

Reply:
`http://listingsca.shoptoit.ca/library/reviews-top-rated-gps/2399` 301s to `http://www.shoptoit.ca/library/reviews-top-rated-gps/2399`.
It's actually the second redirect (to `lt.dell.com`) that times out.